src/components/Global/Footer.js

```diff
@@ -15,23 +15,23 @@ const navigation = [
     name: "SOCIALS",
     href: "/",
     children: [
-      { name: "GitHub", href: "" },
-      { name: "Twitter", href: "" },
-      { name: "Discord", href: "" },
-      { name: "LinkedIn", href: "" },
+      { name: "GitHub", href: "https://github.com/WebXDAO" },
+      { name: "Twitter", href: "https://twitter.com/WebXDAO" },
+      { name: "Discord", href: "https://discord.gg/JaJngac9" },
+      { name: "LinkedIn", href: "https://www.linkedin.com/company/76098793" },
     ],
   },
   {
     name: "CONTACT",
     href: "/contact",
-    children: [{ name: "Contact us", href: "" }],
+    children: [{ name: "Contact us", href: "https://discord.com/invite/TSRwqx4K2v" }],
```

Comment:
We should probably use a mail in this case.
This is the email I found on the CoC.
```suggestion
    children: [{ name: "Contact us", href: "web3opensourcecommunity@gmail.com" }],
```